docs/getting_started_guide/Installing_SIMP_From_A_Repository.rst

```diff
@@ -18,54 +19,55 @@ stack, this is the method that you will almost definitely want to use.
 Enable EPEL
 -----------
 
+.. NOTE::
+
+   RHEL systems will need to enable the `EPEL Repositories`_ manually.
```

Comment:
Do you want to add a code block here to be able to copy/paste an install of the epel-release RPM that corresponds to the system and subsequently installs it? i.e.

    $ yum -y install http://dl.fedoraproject.org/pub/epel/7/x86_64/e/epel-release-7-10.noarch.rpm